src/Fixer/StringNotation/ExplicitStringVariableFixer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\StringNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class ExplicitStringVariableFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Convert implicit variables into explicit ones in double-quoted strings.',
+            [new CodeSample('<?php $a = "My name is $name!";')],
+            'The reasoning behind this rule are the following:'
+                ."\n".'- When there are two valid ways of doing the same thing, using both is confusing, there should be a coding standard to follow'
+                ."\n".'- PHP manual marks "$var" sintax as implicit and "${var}" syntax as explicit: explicit code should always be preferred'
+                ."\n".'- Explicit syntax allows word concatenation inside strings, e.g. "${var}IsAVar", implicit doesn\'t'
+                ."\n".'- Explicit syntax is easier to detect for IDE/editors and therefore has colors/hightlight with higher contrast, which is easier to read'
```

Comment:
@keradus is this way ok?

Reply:
Why not using HEREDOC or a single string, e.g.:
```php
<<<DESC
Foo:
- bar
- baz
DESC
```
or
```php
'Foo:
- bar
- baz'
```
?

Reply:
No essential reason, just dislike to break indentation.
If you prefer, I'll pick one of the two you suggest